blueprints/lib/index.js

```diff
@@ -1,5 +1,22 @@
+var fs = require('fs-extra');
+
 module.exports = {
   description: 'Generates a lib directory for in-repo addons.',
 
-  normalizeEntityName: function(name) { return name; }
+  normalizeEntityName: function(name) { return name; },
+
+  beforeInstall: function() {
+    // make sure to create `lib` directory even if .jshintrc is not created
+    fs.mkdirsSync('lib');
```

Comment:
It should, but it can't because gitkeeps have special handling currently and won't be generated (which is a bug in itself IMHO)